src/store.js

```diff
@@ -63,6 +96,21 @@ export default new Vuex.Store({
     },
     getNextProductId: state => {
       return state.nextProductId;
+    },
+    getCartProductsCount: state => {
+      return state.cart.count;
+    },
+    getCartProducts: state => {
+      let products = []
+
+      state.cart.products.forEach((product) => {
+        products.push({
+          data: state.allProducts.find(item => item.id == product.productId),
+          quantity: product.quantity
+        })
+      })
+
+      return products.length > 0 ? products : false;
```

Comment:
Zamiast zwracać false zwróć pustą tablice. Jak zwrócisz false w `v-for` to mogą się dziwne rzeczy podziać

Reply:
done.